addons/mrp/models/mrp_production.py

```diff
@@ -516,6 +517,18 @@ def _compute_show_lot_ids(self):
         ('qty_positive', 'check (product_qty > 0)', 'The quantity to produce must be positive!'),
     ]
 
+    @api.depends('product_uom_qty', 'date_planned_start')
+    def _compute_forecasted_issue(self):
+        for order in self:
+            warehouse = order.location_dest_id.get_warehouse()
+            order.forecasted_issue = False
+            if order.product_id:
+                virtual_available = order.product_id.with_context(warehouse=warehouse.id, to_date=order.date_planned_start).read(['virtual_available'])[0]["virtual_available"]
```

Comment:
virtual_available = order.product_id.with_context(warehouse=warehouse.id, to_date=order.date_planned_start).virtual_available is doing the same no?